### PR TITLE
Features are added and some code is modified

### PR DESCRIPTION
**Pull request by Bilal-Muhammad**
Files are modified and additional features are added

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -130,4 +130,8 @@ dmypy.json
 
 
 # idea file
-.idea
\ No newline at end of file
+.idea
+
+#ignore environment k liye
+env/
```
Comment (Saleha-Tanveer) [inline .gitignore:136]: Env already included in line 104 gitignore file.

File: Models/NoteModel.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
+from datetime import datetime
+from database import db
+# from run import ma
+
+class Note(db.Model):
+    __tablename__='OneNote'
```
Comment (Saleha-Tanveer) [inline Models/NoteModel.py]: Change table name to "Note".

File: .gitignore (hunk 1 above)
Comment (Bilal-Muhammad) [reply .gitignore:136]: Ok boss ...

**Review by Saleha-Tanveer — APPROVED** [review]
Table name for "Notes" table needs changing.

**Pull request merged**